Homeworks/HW 2/filterArray.js

```diff
@@ -0,0 +1,81 @@
+
+// Task 1 ____________________
+console.log("Task - 1");
+
+function filterArray (numbers, min, max){
+
+    var result = [];
+    var j = 0;
+
+    for (var i = 0; i < numbers.length; i++){
+
+        if (numbers[i] >= min && numbers[i] <= max){
+            result[j] = numbers[i];
+            j++;            
+        }
+
+    }
+    return result;
+}
+
+var someArray = [2, 1, 5, 6, 8, 9, 3, 7, 4, 10, 11];
+console.log("Array before filtering: " + someArray);
+
+var newArray = filterArray(someArray, 3, 6);
+console.log("Array after filtering(from 3 to 6): " + newArray);
+
+// Task 2 ____________________
+console.log("Task - 2");
+
+function isObjectEmpty(obj){
+
+    for(var key in obj){
+        if(obj.hasOwnProperty(key)){
+            return false;
+        }            
+    }
+    return true;
+}
+
+function showMessage (expression){
+    if (expression){
+        console.log("object is empty")
+    }
+    else{
```

Comment:
The else statement should be placed on the same line as the closing curly brace:

if (expression) {
    ...
} else {
    ...
}